### DISCUSSION TITLE
github search provide strange result

### DISCUSSION THREAD

**Discussion by pmakarenko** [community/community#30936] [Other Feature Feedback, Questions, & Ideas]
Hi, I was trying to explore x86-64 At topics and find out that there is repository  
[x86-bare-metal-examples](https://github.com/cirosantilli/x86-bare-metal-examples) having very long сhinese description, it is really long and it is take time to scroll down to the next repo.

Please, add some limits to make searching more comfortable

**Comment by jfullstackdev** [comment]
you can raise an issue there and ask that, so it will not be hard for you to navigate

https://github.com/cirosantilli/x86-bare-metal-examples/issues

**Reply by plovchiks** [reply]
it is not repo issue

**Comment by cirosantilli** [comment]
Related: https://github.com/community/community/discussions/12800
